README.md

```diff
@@ -75,14 +75,14 @@ Julia). Convenience wrapper functions based on Julia types are added as needed.
 **Memory Management**: Programming with SCIP requires dealing with variable and
 constraints objects that use [reference
 counting](https://scip.zib.de/doc-6.0.0/html/OBJ.php) for memory management.
-SCIP.jl provides a wrapper type `ManagedSCIP` that collects lists of `SCIP_VAR*`
+SCIP.jl provides a wrapper type `SCIPData` that collects lists of `SCIP_VAR*`
```

Comment:
Maybe it's better to explain that this type is an implementation detail and that it should not be used by end users directly. Similar to the functions mentioned below (`add_variable` etc).